Finjector.Web/ClientApp/src/pages/Details.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useEffect } from "react";
+import { Link, useParams } from "react-router-dom";
+import FinLoader from "../components/Shared/FinLoader";
+
+import { AeDetails, ChartType } from "../types";
+import { useGetChartDetails } from "../queries/storedChartQueries";
+import { ChartDebugInfo } from "../components/Shared/ChartDebugInfo";
+import { ChartLoadingError } from "../components/Shared/ChartLoadingError";
+import { Alert } from "reactstrap";
+import { renderNameAndEmail } from "../util/util";
+import { ChartNotFound } from "../components/Shared/ChartNotFound";
+import DetailsChartString from "../components/Details/DetailsChartString";
+import PpmDetailsPage from "../components/Details/PpmDetails";
+import { BackLinkBar } from "../components/Shared/BackLinkBar";
+import FinjectorButton from "../components/Shared/FinjectorButton";
+import SharePopup from "../components/Shared/SharePopup";
+import CopyToClipboardHover from "../shared/CopyToClipboardHover";
+
+const Details = () => {
+  const { id, chartSegmentString } = useParams();
+  const chartDetailsQuery = useGetChartDetails(chartSegmentString || "");
+
+  const [chartDetails, setSavedChart] = React.useState<AeDetails>({
+    isValid: false,
+    chartType: "",
+    chartString: chartSegmentString ?? "",
+    chartStringType: ChartType.PPM,
+    errors: [],
+    warnings: [],
+    segmentDetails: [],
+    approvers: [],
+    ppmDetails: {
+      ppmProjectManager: {
+        firstName: null,
+        lastName: null,
+        email: null,
+        name: "",
+      },
+      ppmGlString: "",
+      projectCompletionDate: "",
+      projectStartDate: "",
+      projectStatus: "",
+      projectTypeName: "",
+    },
+    hasWarnings: false,
+  });
+
+  // if we load up new data, update the chart
+  useEffect(() => {
+    if (chartDetailsQuery.data) {
+      const chart = chartDetailsQuery.data;
+      setSavedChart(chart);
+    }
+  }, [chartDetailsQuery.data]);
```

Comment:
dump all of this and replace with `const chartDetails = chartDetailsQuery.data`.  you'll have to add to the error state a condition that `chartDetailsQuery.data` is truthy